modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx

```diff
@@ -12,28 +12,32 @@ import React, {useEffect, useState} from 'react';
 
 import {API_URL, OBJECT_RELATIONSHIP} from '../Constants';
 import {IFDSViewSectionProps} from '../FDSView';
-import OrderableTable from '../components/OrderableTable';
 import openDefaultFailureToast from '../utils/openDefaultFailureToast';
 import openDefaultSuccessToast from '../utils/openDefaultSuccessToast';
+import ActionList from './actions/ActionList';
 import ItemActionForm from './actions/ItemActionForm';
 
 const SECTIONS = {
-	ACTIONS: 'actions',
+	CREATION_ACTIONS: 'creation-actions',
```

Comment:
I've realized that in the backend/DB actions follows the `FDSActions<type>` but in the frontend we are using `<type>-actions` style. We probably need to standardize naming.

Reply:
Not sure I understand, you mean the name of the relationship?

Reply:
@dsanz in `FDSViewsPortlet` we define relationships as `fdsActionCreation|fdsActionItem` and order fields as `fdsActionsCreationOrder|fdsActionsItemOrder`